File: Backend_private/src/main/java/coworking/backend_private/Entidad/Opinion.java
package coworking.backend_private.Entidad;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "OPINIONES")
public class Opinion {

    @Id
    private Integer codigo;
    @ManyToOne(optional = false)
    @JoinColumn(name="codigoCliente")
    private Cliente codigoCliente;
    @ManyToOne(optional = false)
    @JoinColumn(name="codigoReserva")
    private Reserva codigoReserva;
    private String titulo;
    private String opinion;
    private Integer puntuacion;

}

File: Backend_public/app/Models/Opinion.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Opinion extends Model
{
    use HasFactory;

    protected $table = 'OPINIONES';
    protected $primaryKey = 'codigo';
    protected $fillable =['codigoCliente','codigoReserva','titulo','opinion','puntuacion'];
    public $timestamps = false;

}
